pkg/util/selector.go

```diff
@@ -17,7 +17,7 @@ import (
 func ResourceMatches(resource *unstructured.Unstructured, rs v1alpha1.ResourceSelector) bool {
 	if resource.GetAPIVersion() != rs.APIVersion ||
 		resource.GetKind() != rs.Kind ||
-		resource.GetNamespace() != rs.Namespace {
+		(len(rs.Namespace) > 0 && resource.GetNamespace() != rs.Namespace) {
```

Comment:
It means if the Namespace of resourceSelector is empty，the policy can math any resource when they have same APIVersion and Kind?

Reply:
Precisely speaking, `rs.namespace` is a filter, it will be ignored if it is empty, otherwise, it must equal to `rs.Namespace`

Reply:
Got it.